examples/tm4c_led/target.cpp

```diff
@@ -0,0 +1,21 @@
+// Source file contains target-specific implementations.
+
+#include "target.hpp"
+#include <gpio.h>
+#include <pin_map.h>
+
+void gpio_init()
+{
+    // configure UART
```

Comment:
Comment just for your information: tm4c contains  wrappers for pin configuration, example of usage:
https://github.com/forGGe/theCore/blob/develop/tests/targets/tivac_tm4/suites/platform_bat/suite_init.cpp#L11
 
However, since #239 is ongoing, I think there is no need to change this example file to use that wrappers. Possibly that they are will be dropped in favor of build-time, cog-driver configuration